en/sql/identifier.rst

```diff
@@ -49,6 +49,21 @@ Identifiers or reserved keywords shown as below are not allowed. However, if the
     " abc" " def"
     [position]
 
+
+Even identifiers enclosed in double quotation marks, square brackets, or backticks cannot contain square brackets ([ ]) or dots (.).
+
+*  Square brackets are used by default to process reserved words when parsing a query inside.
+*  Dots are used in Path Expressions and JSON, and are also used to separate schema names and object names.
+
+Following are identifiers enclosed in double quotation marks (" ") but not allowed.
+
+::
+
+    "he[[o"
+    "wor]d"
+    "hello[]world"
+    "hello.world"
+
```

Comment:
How about handling warnign visibly?
`.. Warning::`

Reply:
Thanks, I'll change it.